modules/widgets/facebook-likebox.php

```diff
@@ -76,9 +76,21 @@ function widget( $args, $instance ) {
 		$title    = apply_filters( 'widget_title', $instance['title'] );
 		$page_url = set_url_scheme( $like_args['href'], 'https' );
 
-		$like_args['show_faces'] = (bool) $like_args['show_faces'] ? 'true' : 'false';
-		$like_args['stream']     = (bool) $like_args['stream'] ? 'timeline' : 'false';
-		$like_args['cover']      = (bool) $like_args['cover'] ? 'false' : 'true';
+		$like_args['show_faces']   = (bool) $like_args['show_faces'] ? 'true' : 'false';
+		$like_args['stream']       = (bool) $like_args['stream'] ? 'timeline' : 'false';
+		$like_args['cover']        = (bool) $like_args['cover'] ? 'false' : 'true';
+		$like_args['small_header'] = (bool) $like_args['small_header'] ? 'true' : 'false';
+
+		/**
+		 * Filter Facebook Likebox's widget call to action button
+		 *
+		 * @module widgets
+		 *
+		 * @since 8.4.0
+		 *
+		 * @param bool True value hides the call to action button
+		 */
+		$hide_cta = apply_filters( 'jetpack_facebook_likebox_hide_cta', true );
```

Comment:
I would default to `false` here as well, for the same reasons as above.

Reply:
Oh right. Sorry, I checked now and the default is indeed false for both `small_header` and `hide_cta` - https://developers.facebook.com/docs/plugins/page-plugin/ I'll make this change now.